Procfile

```diff
@@ -1,5 +1,4 @@
 web: bundle exec rails s -b 0.0.0.0 -p 3000
 worker: bundle exec rake jobs:work
 mail: mailcatcher -f --smtp-port 2025
-webpack: bin/webpack-dev-server
```

Comment:
In welchem Fall wird dieses Procfile verwendet? Sollte man hier auch noch die Sachen (js & css) vom `Procfile.dev` nachziehen?